patches/spec.fetched.json/07-vlan-metalgateways-href-array-field.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/spec/oas3.patched.json b/spec/oas3.patched.json
+index 0816bf5..fb8fdf0 100644
+--- a/spec/oas3.patched.json
++++ b/spec/oas3.patched.json
+@@ -4549,8 +4549,12 @@
+              },
+              "type": "array"
+            },
+-           "metal_gateway": {
+-             "$ref": "#/components/schemas/MetalGatewayLite"
++           "metal_gateways": {
++             "description": "A list of metal gateways currently associated to this Virtual Network.",
++             "items": {
++               "$ref": "#/components/schemas/Href"
```

Comment:
Reporting this as Href seems fair to me. When `metal_gateways` is included `?include=`, this will be a `MetalGatewayLite`. I don't believe we have simple ways to express that today.